domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1654,7 +1650,7 @@ public TotaledResults(ResultSet set, Integer total) {
 
     vertx.runOnContext(v1 -> {
       try {
-        QueryHelper queryHelper = buildSelectQueryHelper(transactionMode, table, fieldName, where, returnIdField, facets, distinctOn);
+        QueryHelper queryHelper = buildSelectQueryHelper(transactionMode, table, fieldName, where, returnIdField, distinctOn);
```

Comment:
Please also remove the `facets` parameter of `doStreamGet`.

Reply:
Yes.. that's actually RMB-359 resurfacing!!